### ISSUE TITLE
[BUG] Expose load_atomic_multilog through rpc

### ISSUE THREAD

**Issue by haal** [ucbrise/confluo#148]
### Describe the bug
`confluo_store::load_atomic_multilog` is not defined in thrift `rpc_service`

### How to reproduce the bug?
`confluo.rpc.RpcClient` in java is missing method for loading multilog (I assume same is missing for other languages)

### Expected behavior
It is possible to load multilog from java/other rpc client e.g using `RpcClient.loadAtomicMultilog`

**Comment by anuragkh** [comment]
Thanks for the issue; adding support for this in https://github.com/ucbrise/confluo/pull/149.

**Comment by haal** [comment]
I am not sure that loading multilog is working as expected (using latest commit c493592).

Here is the test case:
1. start confluo using data path (e.g. `--data-path=/var/db`)
2. create multilog using java rpc client
```java
RpcClient client = new RpcClient(HOST, PORT);
client.createAtomicMultilog(MULTILOG_NAME, "{ msg: STRING(8) }", StorageMode.DURABLE);
client.disconnect();
```
3. write 1 item to it
```java
RpcClient client = new RpcClient(HOST, PORT);
client.setCurrentAtomicMultilog(MULTILOG_NAME);
client.append("abcdefij");
client.disconnect();
```
4. stop the confluo server 
5. start conflo server using same data path as in step 1
6. loading and trying to get count of items throws exception on server (`Floating point exception`) and stops it
```java
RpcClient client = new RpcClient(HOST, PORT);
client.loadAtomicMultilog(MULTILOG_NAME);
client.setCurrentAtomicMultilog(MULTILOG_NAME);
System.out.println(client.numRecords());
client.disconnect();
```
 - expected behavior in step 6: printed 1

**Comment by anuragkh** [comment]
Sorry I was traveling last week. Will take a look this week.
@ujvl

**Comment by anuragkh** [comment]
Thanks for the report @haal. We've isolated the issue: Confluo's current archival protocol seems to archive at monolog bucket granularity (64MB), so data less than 64MB seems to get lost. We're working on resolving the bug.

**Comment by anuragkh** [comment]
Resolved in https://github.com/ucbrise/confluo/pull/150. Please let me know if this resolves the issue.

**Comment by haal** [comment]
Finally I was able to get some time and verify that initial issue is resolved.

During testing it seems I discovered a new bug and it is related to loading multilog multiple times.

The test case is:
1. start confluo using data path (e.g. --data-path=/var/db)
2. create multilog and write to it
```java
RpcClient client = new RpcClient(HOST, PORT);
client.createAtomicMultilog(MULTILOG_NAME, "{ msg: STRING(8) }", StorageMode.DURABLE);
client.append("abcdefgh");
client.disconnect();
```
3. stop the confluo server
4. start confluo server using same data path as in step 1
5. check count of items - works
```java
RpcClient client = new RpcClient(HOST, PORT);
client.setCurrentAtomicMultilog(MULTILOG_NAME);
System.out.println(client.numRecords());
client.disconnect();
```
6. stop the confluo server
7. start confluo server using same data path as in step 1
8. retry step 5 - it fails with `Floating point exception`

Let me know should I open a separate ticket or we want to handle this as part of this issue?
